Competition2017/tests/competition/subsystems/agitator/AgitatorSubsystemTest.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.agitator;
+
+import org.junit.Test;
+
+public class AgitatorSubsystemTest extends AgitatorTestBase{
+        
+    @Test
+    public void stopTest(){
+        agitator.stop();
+        verifyAgitatorSetpoints(0);
+    }
+    
+    @Test
+    public void spinForwardsTest(){
+        agitator.spinForwards();
+        verifyAgitatorSetpoints(agitator.motorPowerProperty.get());
+    }
+    
+    @Test
+    public void spinBackwardsTest(){
+        agitator.spinBackwards();
+        verifyAgitatorSetpoints(agitator.motorPowerProperty.get() * -1);
+    }
```

Comment:
I suggest at least one more test - 1st, have the agitator go forwards, then stop it. Verify that it go forwards then stops.